packages/apidom-ls/test/openapi-yaml.ts

```diff
@@ -490,10 +725,10 @@ describe('apidom-ls-yaml', function () {
     ];
     assert.deepEqual(result, expected as Diagnostic[]);
 
-    doc = TextDocument.create('foo://bar/specError.yaml', 'yaml', 0, specError);
```

Comment:
I'm not sure I understand this change. Basically this test is no longer running `await languageService.doValidation(doc, validationContext);` at the end. Can you please corroborate why it was removed?

Reply:
There was no followup assertion, so I removed these lines that were leading to it. Note that `openapi-json` doesn't have this.